acl/README.md

```diff
@@ -13,3 +13,47 @@ Binary package
 ## Usage
 
 *TODO: Add instructions for usage*
+
+# Testing acl on Linux
+
+## Build, Install and Test the acl core plan
+
+### Build and Install
+
+Open a terminal at the root directory of the core-plans repo.  Build and install the package:
+
+```bash
+hab pkg build acl
+source ./results/last_build.env
+```
+
+### Test
+
+In a different terminal test the package:
+
+```bash
+source ./results/last_build.env
+hab studio run "./acl/tests/test.sh $pkg_ident"
+```
+
+Sample output from within a studio:
+
+```bash
+[12][default:/src:1]# ./acl/tests/test.sh $pkg_ident
```

Comment:
Sample output should only include the output of the passing tests, not the rest of the test framework output, as so:

```
 ✓ core/acl/2.2.53/20190613191606 has a valid installation directory
 ✓ getfacl binary should exist
 ✓ 'getfacl --version' should equal the plan's version 2.2.53
 ✓ 'getfacl --help' should return valid output
4 tests, 0 failures
```

Reply:
Sample output removed altogether.